plugins/inputs/ceph/README.md

```diff
@@ -133,6 +135,8 @@ All fields are collected under the **ceph** measurement and stored as float64s.
 
 * ceph\_pool\_stats
   * op\_per\_sec (float)
```

Comment:
Add the version of Ceph that removed this field, something like: `* op\_per\_sec (float, ceph < 14)`.